### ISSUE TITLE
Pypy not working with webhooks

### ISSUE THREAD

**Issue by hmdz105** [python-telegram-bot/python-telegram-bot#434]
### Steps to reproduce
1.
Create a simple bot using python-telegram-bot .
2.
Use the following code at the end of your code:

```
updater.start_webhook(listen='0.0.0.0', port=443, url_path=mytoken, 
                           cert='YOURPUBLIC.pem', key='YOURPRIVATE.key', 
                           webhook_url='https://www.mywebsite.com/%s' % mytoken)
```
3.

Run the bot under: 
 A- Python 2.7

Everything is OK. The web-hook is all setup and the bot works smoothly.

 B- Pypy  5.4.1

The following error is raised when the bot is trying to setup the web-hook:

_2016-10-09 01:35:09,746 - telegram.ext.dispatcher - start- CRITICAL - stopping due to exception in another thread
Exception in thread updater:
Traceback (most recent call last):
  File "C:\pypy2-v5.4.1-win32\lib-python\2.7\threading.py", line 806, in __boots
trap_inner
    self.run()
  File "C:\pypy2-v5.4.1-win32\lib-python\2.7\threading.py", line 759, in run
    self.__target(*self.__args, **self.__kwargs)
  File "C:\pypy2-v5.4.1-win32\site-packages\telegram\ext\updater.py", line 117,
in _thread_wrapper
    target(*args, **kwargs)
  File "C:\pypy2-v5.4.1-win32\site-packages\telegram\ext\updater.py", line 281,
in _start_webhook
    self._check_ssl_cert(cert, key)
  File "C:\pypy2-v5.4.1-win32\site-packages\telegram\ext\updater.py", line 315,
in _check_ssl_cert
    raise TelegramError('SSL Certificate invalid')
TelegramError: SSL Certificate invalid_

### Expected behaviour
The start_webhook method shouldn't raise a _SSL certificate invalid exception_ under Pypy [ the same certificate works with Python properly.]

### Actual behaviour

A TelegramError is raised when using Pypy as:

_TelegramError: SSL Certificate invalid_

### Configuration
**Operating System:**
Windows Server 2012 R2 Standard

**Version of Python, python-telegram-bot & dependencies:**

``$ python -m telegram``
python-telegram-bot 5.1.0
urllib3 1.18
certifi 2016.09.26
future 0.15.2
Python 2.7.12 (v2.7.12:d33e0cf91556, Jun 27 2016, 15:19:22) [MSC v.1500 32 bit (
Intel)]

``$ pypy -m telegram``
python-telegram-bot 5.1.0
urllib3 1.18
certifi 2016.09.26
future 0.15.2
Python 2.7.10 (c95650101a99, Sep 06 2016, 15:44:55) [PyPy 5.4.1 with MSC v.1500
32 bit]

*Note:
The same app works properly under pypy when` updater.start_polling(timeout=20)` is used.

**Comment by bomjacob** [comment]
I believe this might be caused by the slight version difference "2.7.12" vs. "2.7.10" due to this line in the changelog:
````- Issue #26930: Update Windows builds to use OpenSSL 1.0.2h.````
Currently not sure if there's any workaround to this issue...

**Comment by hmdz105** [comment]
We have noticed that our bot is more responsive when running via Pypy and using long-polling than running via Python and using webhooks.

And when our bot is using only pure Python, the webhook version is ~30% faster than the long-polling version.

I believe having pypy + webhooks together would be a very powerful combination.

**Comment by hmdz105** [comment]
@bomjacob You were right:

**Issue Resolved** by downgrading Pypy to pypy-5.1.0-win32:

1- Created a Virtualenv targeting the older pypy version [ pypy-5.1.0-win32]
2- Activated the Virtualenv and installed the required dependencies.
3- Ran the Bot under pypy with webhooks enable. Works properly now and the response time is very good.
